src/core/XHPAttributeCoercion.php

```diff
@@ -65,7 +65,7 @@ public static function CoerceToString(
     mixed $val,
   ): string {
     self::LogCoercion($context, 'string', $attr, $val);
-    if (($val is int) || ($val is float) || $val instanceof Stringish) {
+    if (($val is int) || ($val is float) || $val is Stringish) {
```

Comment:
Could be simplified to `if ($val is num || $val is Stringish) {`

Reply:
I found more checks, such as string || arraykey. I simplified them all.